lib/src/renamer.dart

```diff
@@ -183,7 +190,7 @@ class Renamer<T> {
           buffer.writeCharCode(next);
         }
       } else {
-        buffer.writeCharCode(char);
+        buffer.writeCharCode(char!);
```

Comment:
If you make the `if ({...}.contains(char))` check above more explicit you won't need this assertion.